DEVELOPER.md

```diff
@@ -1,188 +1,87 @@
 - [Developer quickstart](#developer-quickstart)
-  - [Code style](#code-style)
-  - [Backend setup](#backend-setup)
-  - [Frontend setup](#frontend-setup)
-  - [Setup using Makefile](#setup-using-makefile)
+  - [Running the project](#running-the-project)
+  - [Running in Docker](#running-in-docker)
+    - [`COMPOSE_PROFILES`](#compose_profiles)
+  - [Useful `make` commands](#useful-make-commands)
+  - [Setting environment variables](#setting-environment-variables)
   - [Slack application setup](#slack-application-setup)
   - [Update drone build](#update-drone-build)
-- [Troubleshooting](#troubleshooting)
-  - [ld: library not found for -lssl](#ld-library-not-found-for--lssl)
```

Comment:
I would imagine some of these could be still useful when `engine` is running not in docker.
For example, I couldn't get my PyCharm connected to the Python interpreter in the engine service, so for now I would probably use docker-compose without the `engine` profile, so I can connect to the local interpreter for debugging purposes.